docker-compose.yml

```diff
@@ -62,6 +62,18 @@ services:
      - "127.0.0.1:8060:8060"
     privileged: true
 
+  snekbox-311:
+    << : *logging
+    << : *restart_policy
+    image: ghcr.io/python-discord/snekbox:3.11
+    init: true
+    ipc: none
+    ports:
+     - "127.0.0.1:8065:8060"
+    privileged: true
+    profiles:
+      - "3.11"
```

Comment:
Because the eval command defaults to 3.11, it would make more sense for the 3.11 service to start by default and for the 3.10 service to be opt-in.

Reply:
[`e5006b2` (#2218)](https://github.com/python-discord/bot/pull/2218/commits/e5006b291fbb2cfe96dab0c34cf1254f7795d3b0)

Reply:
Hmm I don’t quite like this solution. The 3.11 image is ~2x the size because of the dependencies. Could we make the default an environment setting?

Reply:
Yea, we could make it default to 3.10 in dev to avoid contribs having to clone the hug image, and then have it be 3.11 in prod.

Reply:
Alternatively, we could build and push a 3.11 image without deps.

Reply:
Updated to use the 3.11 dev image in this commit https://github.com/python-discord/bot/pull/2218/commits/faea2027414beebf5d1ee75c0b0c8af4a0c94cf3 this reduces the image size down to the normal snekbox image.